tests/bsim/bluetooth/ll/bis/src/main.c

```diff
@@ -148,6 +148,53 @@ bool ll_data_path_sink_create(uint16_t handle, struct ll_iso_datapath *datapath,
 }
 #endif /* CONFIG_BT_CTLR_ISO_VENDOR_DATA_PATH */
 
+#define BUF_ALLOC_TIMEOUT_MS (30) /* milliseconds */
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU), 8, NULL);
+
+static struct k_work_delayable iso_send_work;
+
+BUILD_ASSERT(sizeof(seq_num) <= CONFIG_BT_ISO_TX_MTU);
+
+static void iso_send(struct k_work *work)
+{
+	static uint8_t iso_data[CONFIG_BT_ISO_TX_MTU] = { 0, };
+	static bool data_initialized;
+	struct net_buf *buf;
+	size_t iso_data_len;
+	int ret;
+
+	if (!data_initialized) {
+		data_initialized = true;
+
+		for (int i = 0; i < ARRAY_SIZE(iso_data); i++) {
+			iso_data[i] = (uint8_t)i;
+		}
+	}
+
+	buf = net_buf_alloc(&tx_pool, K_MSEC(BUF_ALLOC_TIMEOUT_MS));
+	if (!buf) {
+		FAIL("Data buffer allocate timeout on channel\n");
+		return;
+	}
+
+	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
+	sys_put_le16(seq_num, iso_data);
+	iso_data_len = MAX(sizeof(seq_num), ((seq_num % CONFIG_BT_ISO_TX_MTU) + 1));
```

Comment:
Why the `MAX? Couldn't we just do
```suggestion
	iso_data_len = ((seq_num % CONFIG_BT_ISO_TX_MTU) + 1);
```

Reply:
2 byte `seq_num` is always to be sent. And want the length in the range 2 to 251 i.e. `((250 % 251) + 1)`